codegen/elm.codegen.json

```diff
@@ -10,6 +10,6 @@
             "elm/url": "1.0.0",
             "dillonkearns/elm-pages": "10.1.0"
         },
-        "local": []
+        "local": ["helpers"]
```

Comment:
This is cool! I was not aware this was possible!

Reply:
It is! The main downside is that we now have the definitions twice, but the new version of `elm-codegen` is going to fix that too :eyes: